src/test/integration/integrationtests.py

```diff
@@ -6,9 +6,37 @@
 
 class gateway_tests(unittest.TestCase):
 
-	# Tests that a good request returns a 200
-	def test_success(self):
-		self.assertEqual(good_request(url, access_token), 200)
+	# Tests a single resource ID in different case styles
+	def test_id(self):
```

Comment:
That's odd. I thought I had set Sublime to do spaces, and it did for api_request.py but not the other files. I'll update those.